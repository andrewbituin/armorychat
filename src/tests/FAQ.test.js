import React from 'react';
import FAQ from '../Components/Faqs/FAQ';
import ReactDOM from 'react-dom';
import { MemoryRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';

describe('FAQ component', () => {

  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(
      <MemoryRouter>
        <FAQ />
      </MemoryRouter>, div
    );
    ReactDOM.unmountComponentAtNode(div);
  })

  it("renders the UI as expected", () => {
    const tree = renderer
      .create(
        <MemoryRouter>
          <FAQ />
        </MemoryRouter>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
})