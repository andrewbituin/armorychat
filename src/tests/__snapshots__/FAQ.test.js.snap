// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FAQ component renders the UI as expected 1`] = `
<div
  className="faq-container"
>
  <h1>
    Frequently Asked Questions
  </h1>
  <ul>
    <li>
      What is Squad Armory?
    </li>
    <p>
      Squad Armory is a website designed to help individuals find groups of people to play games with. We know that playing games online, by yourself, or with random teams can get frustrating. Squad Armory wants to help you have fun and find your squad.
    </p>
    <li>
      How do I use your site?
    </li>
    <p>
      Using our site is easy. Sign up to view games, and available parties. You must confirm your email if you wish to join or create a party. Once youve joined a party, you can participate in a chat between your squad-mates.
    </p>
    <li>
      Will you add more games?
    </li>
    <p>
      We will add games that we feel are trending at the time. If you have a title that you would like to see listed, feel free to contact us.
    </p>
    <li>
      Why should I use this service rather than Discord or Twitch?
    </li>
    <p>
      You shouldn't. Squad Armory isn't trying to replace the need for such services. In fact, the party chat was put in place so that you could share usernames and decide on a venue for your in game chat. We are simpily here to make finding friends easy.
    </p>
    <div
      className="faq-footer"
    >
      <li>
        Need to ask a question?
      </li>
      <a
        href="mailto:armorysquad@gmail.com"
      >
        Contact Us
      </a>
    </div>
    <p />
    <button
      onClick={[Function]}
    >
      Back
    </button>
  </ul>
</div>
`;
